### DISCUSSION TITLE
RegEx [Jul 21]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2870] [Discord Threads]
> 🙋‍♂️ **rohitstartsup**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Not supported currently  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/999310237326647358)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
